core/static/core/scss/core.scss

```diff
@@ -101,6 +101,7 @@ article {
     top: 0;
     // padding: 1em 0;
     position: sticky;
+    z-index: 3;
```

Comment:
Maplibre seems to add a higher z-index to maps by default, so making sure that the date headers (e.g. "mar 30 (sun)") scroll over the maps instead of below